Experimental/MultiplayerUseCases/Assets/UseCaseSamples/NetvarVsRpc/Scripts/ColorManager.cs

```diff
@@ -65,23 +65,23 @@ void Update()
 
         void OnClientRequestColorChange()
         {
-            OnServerChangeColorServerRpc();
+            ServerChangeColorRpc();
         }
 
-        [ServerRpc(RequireOwnership = false)] //note: please refer to RPCs documentation to learn more about the pros and cons of the RequireOwnership parameter
-        void OnServerChangeColorServerRpc()
+        [Rpc(SendTo.Server, RequireOwnership = false)] //note: please refer to RPCs documentation to learn more about the pros and cons of the RequireOwnership parameter
```

Comment:
I see RequireOwnership used in several Rpcs, when its default value is false. We should call attention to its use when it's set to true in one of our example scenes.

Reply:
So you do mean we should remove it here, as its default value is false by default?

Reply:
Right. Let's remove redundant code primarily, and draw attention to this flag when we need to set it to true.

Reply:
I just wonder in this scenario the comment after also let's user check out the pros and cons of RequireOwnership, which would still be good to have. And if the comment stands there without the code beforehand it might be a bit confusing.

Reply:
That would be great to point out when it fits the use-case.
At the moment it's adding noise, and also adding redundant code to the attribute definition.

Reply:
I agree with Fernando, that code doesn't add anything useful at the moment.

Reply:
That is fair. I am not sure if we have the use-case somewhere else (cannot recall any). But you are right, redundant code should not just be added for the sake of linking to extra information. I am gonna remove it here.